scripts/ci.sh

```diff
@@ -37,6 +37,21 @@ aws --version
 echo "Initializing modules..."
 terraform init
 
+echo "Validating modules ..."
```

Comment:
A couple of things. Leaving it to you to think about. Personally, I really prefer to put the various parts of this script into the Makefile. This way they are runnable in isolation which is good for development and local testing of each step. The script itself is often than just used to prepare the "in pipeline" specific environment (aka setting env variables etc). See also https://github.com/jenkins-x/terraform-google-jx/blob/master/scripts/ci.sh.

So I would put the installation and execution of tfsec as two rules into the Makefile, especially since this seems to be a static analysis and the cluster is not needed for that.

My second point is a continuation to that. This tfsec is independent of the shellspec tests we could add these tests as dedicated steps to _jenkins-x.yml_ (especially easy if tfsec is integrated into the Makefile). The benefit is that the reporting is nicer. We have dedicated logs for each step opposed to single log for the execution of _ci.sh_ and one needs to figure out what all the output means. 

WDYT?

Reply:
Makefile makes more sense. Putting it into jenkins-x.yml would be cool!
I also feel it's time to move to terratests/kitchen sink for running the tests: https://github.com/jenkins-x/terraform-aws-eks-jx/issues/30

The issue with not being able to see the logs (iirc, only cloudbees employees can see it) makes it a bit difficult for me to suggest changes to the pr build checks (I am blind to how the logs look in the pipeline steps) :)